### DISCUSSION TITLE
Packages that fail to build on the current version.

### DISCUSSION THREAD

**Discussion by Spamm00r** [Freetz-NG/freetz-ng#840] [Unerfüllte Wünsche] [answered]
Hi,

the current version fails to build the following packages:
GOCR 
MINIDLNA
Indanyy
NMAP
opendd
sipproxd
python2
sms3tools
vtun
Polarssl 

The reasons range from freetz failing to locate the source for example for polarssl, which you can fix by manually downloading them, to any other reasons that break the build process.

I had to disable them to build freetz successfully.

Please fix.

Thanks!

**Comment by fda77** [comment, marked as answer]
No, feel free to send PRs
